src/components/Table/TableBody/RowActionsCell/RowActionsCell.jsx

```diff
@@ -181,12 +181,14 @@ class RowActionsCell extends React.Component {
                     onOpen={this.handleOpen}
                     onClose={this.handleClose}
                     className={`${iotPrefix}--row-actions-cell--overflow-menu`}
+                    selectorPrimaryFocus={`.${iotPrefix}--action-overflow-item--initialFocus`}
```

Comment:
Whats the difference between this and the `primaryFocus` prop ?

Reply:
This is the new way Carbon is doing focus for the OverflowMenu